code_samples/additional-info/reactions/raw-event.js

```diff
@@ -15,7 +15,16 @@ client.on('raw', async event => {
 
 	const user = client.users.get(data.user_id);
 	const message = await channel.fetchMessage(data.message_id);
-	const reaction = message.reactions.get(data.emoji.id || data.emoji.name);
+
+	let emoji = ''
+
+	if(data.emoji.id) {
+		emoji = `${data.emoji.name}:${data.emoji.id}`
+	} else {
+		emoji = data.emoji.name
+	}
+
+	const reaction = message.reactions.get(emoji);
```

Comment:
I think a ternary could be used here, just to keep the LOC relatively short and simple. Something like this is what I had in mind:

```js
const emojiKey = (data.emoji.id) ? `${data.emoji.name}:${data.emoji.id}` : data.emoji.name;
const reaction = message.reactions.get(emojiKey);
```

Apply this to the guide page explanation as well, of course.